assets/app/components/site/siteBuildLogs.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import buildLogActions from "../../actions/buildLogActions";
+
+class SiteBuildLogs extends React.Component {
+  componentDidMount() {
```

Comment:
I would prefer we move away from fetches triggered from view lifecycle actions, but we repeat this idiom many times in the application.

It's better to have the data load call not part of the view so the view is just a view of given state. As such, this view will flash with a "no build logs" until it gets the data from the Federalist API.

I'm definitely not going to hold back this merge for a change because we have accepted this design pattern in the client many times.